CMakeLists.txt

```diff
@@ -358,6 +355,20 @@ else()
       POSITION_INDEPENDENT_CODE true)
 endif()
 
+
+# Now target jansson is declared, set per-target values
+
+target_compile_definitions(jansson PUBLIC JANSSON_USING_CMAKE)
+target_compile_definitions(jansson PRIVATE HAVE_CONFIG_H)
+
+target_include_directories(jansson PUBLIC
+	$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
+	INTERFACE $<INSTALL_INTERFACE:include>
+)
+
+add_library( Jansson::jansson ALIAS jansson )
```

Comment:
This change is unrelated to the topic of this PR. An alias is often added for consistency so that the target name is the same whether you use `add_subdirectory`, `FetchContent`, or `find_package`.

Reply:
It should be `jansson::jansson` with a lowercase j, to be consistent with the exported namespace.